gnomic/models.py

```diff
@@ -126,6 +126,19 @@ def match(self, other):
         return all(a.match(b) for a, b in zip(self.contents, other.contents))
 
 
+class MarkerSet(FeatureTree, MatchableMixin):
```

Comment:
Is there a reason to have a MarkerSet as opposed to just a normal set? If it is required for matching, it should still be possible to initialize a feature with a normal list/set in the same way you can initialize a Mutation with a normal list of changes.